test/latest/Dockerfile.arm64

```diff
@@ -56,6 +56,14 @@ FROM test-git as test-git-lfs
 # renovate: datasource=github-releases packageName=git-lfs/git-lfs
 RUN install-tool git-lfs v3.4.1
 
+#--------------------------------------
+# Image: gleam
+#--------------------------------------
+FROM base as test-gleam
+
+# renovate: datasource=npm
+RUN install-tool gleam 0.34.1
+
```

Comment:
revert, wrong file. add to Dockerfile.jammy

Reply:
[eb0f485](https://github.com/containerbase/base/pull/2018/commits/eb0f485c857bdb4c9a99270730d5ca9d17c9d706)